From 6d7aa31dfde1bd38aa65ef4434930a9fa5b0b5e1 Mon Sep 17 00:00:00 2001
From: Matthias Schoth <mschoth@gmail.com>
Date: Mon, 25 May 2020 02:30:41 +0200
Subject: [PATCH] Trivial patch that reduces input/output latency by
 selectively disabling the built in frame rate limiting for KeyPress events.

---
 x.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/x.c b/x.c
index e5f1737..f6b7604 100644
--- a/x.c
+++ b/x.c
@@ -1933,6 +1933,8 @@ run(void)
 			dodraw = 1;
 			last = now;
 		}
+		if (ev.type == KeyPress)
+			dodraw = 1;
 
 		if (dodraw) {
 			while (XPending(xw.dpy)) {
-- 
2.26.2

